apps/gallery/test/marionette/delete_image_test.js

```diff
@@ -19,37 +19,117 @@ marionette('using the gallery menu', function() {
   });
 
   setup(function() {
-    // Remove all files in temp device storage.
-    client.fileManager.removeAllFiles();
-    // Add file into the pictures directory
-    client.fileManager.add({
-      type: 'pictures',
-      filePath: 'test_media/Pictures/firefoxOS.png'
-    });
     app = new Gallery(client);
-    app.launch();
   });
 
-  test('should let you select images', function() {
-    // Use the 'selection' button and select the test image.
-    // Selected images should have an border/highlight.
-    app.thumbnailsSelectButton.click();
-    app.thumbnail.click();
+  suite('Gallery app with single image', function() {
+    setup(function() {
+      // Remove all files in temp device storage.
+      client.fileManager.removeAllFiles();
+      // Add file into the pictures directory
+      client.fileManager.add({
+        type: 'pictures',
+        filePath: 'test_media/Pictures/firefoxOS.png'
+      });
+    });
+
+    suite('Select and delete single image', function() {
```

Comment:
Good feedback, will update to test single delete image by starting app with three images.